src/main/java/uk/gov/hmcts/reform/sscs/model/servicebus/SessionAwareServiceBusMessagingService.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.sscs.model.servicebus;
+
+import com.azure.messaging.servicebus.ServiceBusMessage;
+import com.azure.messaging.servicebus.ServiceBusSenderClient;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.hmcts.reform.sscs.service.servicebus.SessionAwareMessagingService;
+
+@Slf4j
+public class SessionAwareServiceBusMessagingService implements SessionAwareMessagingService {
+
+    private final ServiceBusSenderClient senderClient;
+
+    public SessionAwareServiceBusMessagingService(
+        ServiceBusSenderClient senderClient) {
+        this.senderClient = senderClient;
+    }
+
+    @Override
+    public boolean sendMessage(SessionAwareRequest message) {
+
+        try {
+            ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
suggest moving this up to the class level - it's thread safe so can be static final. in such way new instance won't be created every time this method is invoked